### ISSUE TITLE
Can't extract Picard CollectRnaSeqMetrics 

### ISSUE THREAD

**Issue by ewels** [ewels/MultiQC#415]
Moved from ewels/MultiQC_TestData#30 :

> Hi Phil, 
> 
> As we discussed over Gitter, I have trouble to get MultiQC recognize the CollectRnaSeqMetrics results. Please find the attached an example file for you to test. Thanks!
> 
> Best regards,
> Lei
> [CTC-1-AA-B2.star.bam.rnaseqmetrics.txt](https://github.com/ewels/MultiQC_TestData/files/815430/CTC-1-AA-B2.star.bam.rnaseqmetrics.txt)

**Comment by ewels** [comment]
From [Gitter](https://gitter.im/ewels/MultiQC?at=58b88840872fc8ce62aa0be0):

> For some reason the result does not appear in the final MultiQC report even though the standard output of the program shows otherwise (`[INFO ] picard : Found 21 RnaSeqMetrics reports`).  I'm curious about how MultiQC searches the file and why it is missing in the report. Thanks in advance for your help! -Lei

**Comment by ewels** [comment]
Hi @lxhuang7,

As I guessed - the file you sent looks a bit different from the examples that I used to write the module. The main difference seems to be the text case for the strings that MultiQC uses to find and parse the file - the example I had used `CollectRnaSeqMetrics` and `RnaSeqMetrics` but your file has `Collectrnaseqmetrics` and `rnaseqmetrics`. For my installation of MultiQC your file isn't actually found at all. This is a bit worrying, as most of the Picard module names use CamelCase so the same problem could affect all of them. What version of Picard are you using?

The second problem was that your file had two lines of stats - one for everything, and one for a named read group (which was also everything). This broke the histogram, though the General Stats table was fine. I've updated the code to handle this - it now takes the first line of stats and then parses the histogram with the first column of data (`All_Reads.normalized_coverage`).

Anyway, should all be fixed now..

Phil
